pypesto/select/misc.py

```diff
@@ -36,6 +38,8 @@ def model_to_pypesto_problem(
         Values in `x_guess` for parameters that are not estimated will be
         ignored and replaced with their value from the PEtab Select model, if
         defined, else their nominal value in the PEtab parameters table.
+    hierarchical:
+        Whether the problem involves hierarchical optimization.
```

Comment:
How about directly forwarding any `**kwargs` to `PetabImporter()`?

Reply:
Could be done as well. 
I personally found it more transparent in this way, instead of hiding it in `**kwargs`.

Reply:
These methods (`model_to_pypesto_problem` and `model_to_hierarchical_pypesto_problem`) are used as an argument [1].

A separate method was provided so that the user doesn't need to setup with `partial` like
```python
from functools import partial
model_to_pypesto_problem_method = partial(model_to_pypesto_problem, hierarchical=True)
pypesto_select_problem.select_to_completion(..., model_to_pypesto_problem_method=model_to_pypesto_problem_method)
```

I think forwarding `**kwargs` is a good idea, but they will need to be forwarded to two calls: `correct_x_guesses` and `PetabImporter`

[1] https://github.com/ICB-DCM/pyPESTO/blob/b4a187f8ac10ce0cc12a22f49cd75325f3811d8c/pypesto/select/method.py#L250